charts/umbrella/README.md

```diff
@@ -344,6 +368,24 @@ helm delete umbrella --namespace umbrella
 > If persistance for one or more components is enabled, the persistent volume claims (PVCs) and connected persistent volumes (PVs) need to be removed manually even if you deleted the release from the cluster.
 >
 
+### Database Access
+
+This chart also contains a pgadmin4 instance for easy access to the deployed Postgres databases which are only available from within the Kubernetes cluster.
+
+pgadmin4 is by default enabled with in the predefined subsets for data exchange and portal.
+
+Address: [pgadmin4.tx.test](http://pgadmin.tx.test)
```

Comment:
I think it should be either `pgadmin4.tx.test` or `pgadmin.tx.test`. Not the text and link are different.

Reply:
thanks, I corrected it